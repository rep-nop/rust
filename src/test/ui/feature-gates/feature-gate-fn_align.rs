#![crate_type = "lib"]

#[repr(align(16))] //~ ERROR the attribute `repr(align(...))` on functions is unstable
fn requires_alignment() {}
