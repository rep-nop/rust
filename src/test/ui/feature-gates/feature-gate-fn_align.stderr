error[E0658]: the attribute `repr(align(...))` on functions is unstable
  --> $DIR/feature-gate-fn_align.rs:3:8
   |
LL | #[repr(align(16))]
   |        ^^^^^^^^^
   |
   = note: see issue #82232 <https://github.com/rust-lang/rust/issues/82232> for more information
   = help: add `#![feature(fn_align)]` to the crate attributes to enable

error: aborting due to previous error

For more information about this error, try `rustc --explain E0658`.
